packages/mux-player/src/media-theme-mux/styles.css

```diff
@@ -98,35 +98,34 @@ media-controller {
 
 :host media-time-range {
   color: var(--_primary-color);
+  --media-range-thumb-opacity: 0;
 }
 
 :host(:not([audio])) media-time-range {
   --media-range-padding-left: var(--mux-time-range-padding, 10px);
   --media-range-padding-right: var(--mux-time-range-padding, 10px);
+  --media-range-padding: 0;
   width: 100%;
-  z-index: 10;
-  height: 10px;
-  bottom: -2px;
-  background: linear-gradient(
-    180deg,
-    transparent,
-    transparent 3px,
-    var(--media-control-background) 3px,
-    var(--media-control-background) 8px,
-    transparent 8px,
-    transparent
-  );
```

Comment:
this was still a left over of when we made the time range bigger for getting more hover area.
it's now built-in Media Chrome so we can remove it here.

Reply:
this is put back in to be able to have a better experience on mobile having a higher time range bar